### DISCUSSION TITLE
Add utilities like mdUpper, lgUpper

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [chakra-ui/panda#2708] [Ideas]
I use a lot of mdDown, and lgDown. But there is not something in the opposite.

**Comment by segunadebayo** [comment]
mdUpper, and lgUpper sound like the current `md` and `lg` breakpoints.

**Reply by juliomuhlbauer** [reply]
Makes sense, it was not intuitive for me though. I thought that md was applying only to md screens, but makes sense because there is a mdToLg.
I think it would be more intuitive to use mdUpper instead.
Thanks!

**Reply by segunadebayo** [reply]
By design, you really shouldn't have to rely on `*Down` and `*Up` (proposed) if you use the mobile-first approach.

We won't be able to support this right away since there's an existing solution. I admit it might not be intuitive for you.
